R-package/pkgdown/_pkgdown.yml

```diff
@@ -0,0 +1,93 @@
+url: https://github.com/dmlc/xgboost
+
+template:
+  bootstrap: 5
+  math-rendering: mathjax
+
+reference:
```

Comment:
Could the CI job be configured in such a way that it'd throw a warning if some function is exported but not listed in this file?

Reply:
Pkgdown throws an error if there's a missing symbol.